### PR TITLE
Author pages

### PR DESCRIPTION
**Pull request by jkpl**
* A page that lists all team members
* Improved page template for customized homepage

### CODE REVIEW

**Comment by jkpl** [comment]
TODO:
* Styling
* Descriptions for each team member
* Link from homepage

File: resources/config.edn [hunk 1]
```diff
@@ -10,4 +10,17 @@
              :description "<b>Yet Another Hints Extension</b> is an extension for Chrome that allows users to activate links without touching the mouse."
              :github-id "yahe"}]
  :authors {:jkpl {:firstname "Jaakko"
-                  :lastname "Pallari"}}}
+                  :lastname "Pallari"
```
Comment (jkpl) [inline resources/config.edn]: EDN standard formatting is horrible for this.

**Comment by jkpl** [comment]
Currently there's no specific style used in the team page or member pages. @thevangelist could we get your input for the style and structure for those pages?.

**Comment by thevangelist** [comment]
I can check the styles, I'll probably do a SCSS rewrite with BEM #8

**Comment by jkpl** [comment]
@thevangelist cool, thanks!

**Pull request merged**